.github/workflows/codeql.yml

```
@@ -20,14 +20,25 @@ jobs:
           languages: cpp, java
 
       - name: Install Dependencies
-        run:  
-          sudo apt-get -y update
+        run: sudo apt-get -y update
+
+          sudo apt-get -y install ninja-build extra-cmake-modules scdoc
 
-          sudo apt-get -y install ninja-build extra-cmake-modules scdoc qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools libqt5core5a libqt5network5 libqt5gui5 libqt5networkauth5 libqt5networkauth5-dev libqt5opengl5 libqt5opengl5-dev
+      - name: Install Qt (macOS, Linux, Qt 6 & Windows MSVC)
+        uses: jurplel/install-qt-action@v3
+        with:
+          aqtversion: "==3.1.*"
+          py7zrversion: ">=0.20.2"
+          version: "6.7.0"
```

```suggestion
          version: "6.8.1"
```

This should be kept in sync with what we're currently using on Linux in CI